openQA.spec

```diff
@@ -45,23 +45,31 @@
+# The following line is generated from dependencies.yaml
 %define client_requires curl git-core jq perl(IO::Socket::SSL) >= 2.009 perl(IPC::Run) perl(JSON::Validator) perl(LWP::Protocol::https) perl(LWP::UserAgent) perl(YAML::PP) >= 0.020 perl(YAML::XS)
-%define worker_requires openQA-client optipng os-autoinst < 5 perl(Minion::Backend::SQLite) perl(Mojo::IOLoop::ReadWriteProcess) > 0.19 perl(Mojo::SQLite)
+# The following line is generated from dependencies.yaml
+%define worker_requires openQA-client optipng os-autoinst < 5 perl(Minion::Backend::SQLite) perl(Mojo::IOLoop::ReadWriteProcess) >= 0.20 perl(Mojo::SQLite)
+# The following line is generated from dependencies.yaml
 %define build_requires %assetpack_requires rubygem(sass)
 
 # All requirements needed by the tests executed during build-time.
 # Do not require on this in individual sub-packages except for the devel
 # package.
-%define test_requires %common_requires %main_requires %python_scripts_requires %worker_requires ShellCheck curl jq os-autoinst-devel perl(App::cpanminus) perl(Perl::Critic) perl(Perl::Critic::Freenode) perl(Selenium::Remote::Driver) perl(Selenium::Remote::WDKeys) perl(Test::Fatal) perl(Test::MockModule) perl(Test::Mojo) perl(Test::More) perl(Test::Output) perl(Test::Pod) perl(Test::Strict) perl(Test::Warnings) postgresql-server python3-setuptools python3-yamllint
+# The following line is generated from dependencies.yaml
+%define test_requires %common_requires %main_requires %python_scripts_requires %worker_requires ShellCheck curl jq os-autoinst-devel perl(App::cpanminus) perl(Perl::Critic) perl(Perl::Critic::Freenode) perl(Perl::Tidy) == 20200110 perl(Selenium::Remote::Driver) >= 1.23 perl(Selenium::Remote::WDKeys) perl(Test::Fatal) perl(Test::MockModule) perl(Test::Mojo) perl(Test::More) perl(Test::Output) perl(Test::Pod) perl(Test::Strict) perl(Test::Warnings) postgresql-server python3-setuptools python3-yamllint
```

Comment:
I see a (minor) problem here. You add Perl::Tidy which should IMHO only be required in `devel_requires` as we exclude any tidy check in the OBS tests

Reply:
ok, Perl::Tidy was defined in `cpanfile` under `test`. Guess it's time to add a `devel` target also in that file.

Reply:
done